mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workload_test.go

```diff
@@ -0,0 +1,149 @@
+package apphub_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccDataSourceApphubDiscoveredWorkload_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":          envvar.GetTestOrgFromEnv(t),
+		"random_suffix":   acctest.RandString(t, 10),
+		"billing_account": envvar.GetTestBillingAccountFromEnv(t),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testDataSourceApphubDiscoveredWorkload_basic(context),
+			},
+		},
+	})
+}
+
+func testDataSourceApphubDiscoveredWorkload_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_project" "service_project" {
+	project_id ="tf-test-ah-%{random_suffix}"
+	name = "Service Project"
+	org_id = "%{org_id}"
+	billing_account = "%{billing_account}"
```

Comment:
indentation should be two spaces for configurations.

Reply:
Done.